packages/apps/graph/generated-schema.graphql

```diff
@@ -290,7 +290,7 @@ type NonFungibleTokenBalance {
   balance: Int!
   chainId: String!
   guard: Guard!
-  id: ID!
+  id: String!
```

Comment:
This should probably become `tokenId`. Since the id property in GraphQL is what we use as the Relay node id.

Reply:
Yep, agree. Will change it

Reply:
I'm not sure if there are other object definitions where a DB id and a Relay ID possibly clash. Might have to check that

Reply:
These will imply changes in `graph-client` and PoU

Reply:
Should we merge this as is, and put those changes on a different PR to be dealt with when we have a proper way of dealing with breaking changes? @MRVDH @alber70g 
Or should I do it in this PR and have it on hold

Reply:
Let's put it in a different PR